### ISSUE TITLE
User can click on, search for, route to Commercial Overlays. Details link to handbook info. 

### ISSUE THREAD

**Issue by andycochran** [NYCPlanning/labs-zola#309]
Need to get some summary text so we can display it on the route.

**Comment by andycochran** [comment]
Use the tooltip for the summary text. Unblocking…

**Comment by andycochran** [comment]
@allthesignals I think we should close this as wontfix. You can already get to Commercial Overlay details from the lot view. We need to establish rule for why something is routable. I don't think "so you can link externally" is a good reason for something to be routable. Special Purpose districts are routable because you can search for them, which is a good reason. Objections to closing this?

**Comment by fruchala** [comment]
It’s weird to me that people would be able to search for nearly every zoning district except for C1 and c2 overlays which are some of the most important in the ZR. Why this one as opposed to any of the others?

Sent from my iPhone

On Oct 20, 2017, at 10:02 AM, Andy Cochran <notifications@github.com<mailto:notifications@github.com>> wrote:


@allthesignals<https://github.com/allthesignals> I think we should close this as wontfix. You can already get to Commercial Overlay details from the lot view. We need to establish rule for why something is routable. I don't think "so you can link externally" is a good reason for something to be routable. Special Purpose districts are routable because you can search for them, which is a good reason. Objections to closing this?

**Comment by andycochran** [comment]
Thanks, @fruchala. That's a good reason for making COs routable AND searchable.

**Comment by allthesignals** [comment]
How should routing working for this? Is it feature-level or...?

**Comment by allthesignals** [comment]
![Screen Shot 2017-11-20 at 1.48.50 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5830e3bc40b5a00e00a5b4c0/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b76226456f36b7191a379f116234f5a5f5ecc3c4a4356bcbbd671f3e57862804e0ee4836d5b7d903a17a9ea8149e901ba21d94684b7eddb4dd2f42f67b6d5db1063474f4dcbed5727a29941c95df74ae34d56a3173e6ae47e2431.png)

Any thoughts on the language for this view? I'll just link to commercial overlay?
